caddyhttp/httpserver/context.go

```diff
@@ -57,6 +58,24 @@ func (c Context) Header(name string) string {
 	return c.Req.Header.Get(name)
 }
 
+// Env gets a map of the environment variables.
+func (c Context) Env() map[string]string {
```

Comment:
I thought the same thing, but other go templating uses the map notion. So keeping it the same might be good for portability and the 'barer to entry', so to speak. Perhaps caching the map up front on Context creation to so avoid processing on each call?